packages/hooks/src/usePlatformRealAccounts.ts

```diff
@@ -0,0 +1,29 @@
+import { useStore } from '@deriv/stores';
+
+/**
+ * we can use this hook to get all real accounts for both Eu and Non-Eu regions.
+ * it loops through the accounts list and returns all real accounts
+ * @example const real_accounts = usePlatformRealAccounts();
+ * @returns [{ balance: 100, currency: 'USD' }, { balance: 50, currency: 'EUR' }]
+ */
+
+const usePlatformRealAccounts = () => {
+    const { client, traders_hub } = useStore();
+    const { accounts } = client;
+    const { is_eu_user } = traders_hub;
+    const account_list = Object.keys(accounts).map(loginid => accounts[loginid]);
+
+    const platform_real_accounts = account_list.filter(account => {
+        const is_maltainvest =
+            'landing_company_shortcode' in account && account.landing_company_shortcode === 'maltainvest';
```

Comment:
@shayan-deriv `landing_company_shortcode` doesn't seem to exist on the `account` type, Are we sure this code is correct? 🤔

Reply:
@farzin-deriv it exists : 
![image](https://user-images.githubusercontent.com/100833613/231058863-2bb0cf9e-3fdf-4e52-8a44-77c5fd4228ff.png)
the old code also is like this otherwise the total balance foer EU / Non-EU account doesn't calculate correctly